### DISCUSSION TITLE
How do I change the amount of threads the virtual machine has?

### DISCUSSION THREAD

**Discussion by Its3rr0rsWRLD** [utmapp/UTM#3697] [UTM Help (macOS)]
**How do I change it?**

If I try to add `cpus=10,sockets=1,cores=10,threads=6` to QEMU, it just crashes and says,

`gemu-aarch64-softmmu:
cpus=10,sockets=1,cores=10,threads=6:
Could not open
cpus=10,sockets=1,cores=10,threads=6'
: No such file or directory`

If I take it off, it runs fine but its maxed out at 1GHZ :(

**Configuration**
3.0.4
12.2.1
M1 Max

**Upload VM**
https://drive.google.com/file/d/1X4ClE01zUK4oBgmphoSXj1DkO_CEPf1n/view?usp=sharing

**Video**
Btw this will download the video. I had to convert from `mkv` to `mp4`.
https://storage.cloudconvert.com/tasks/8b7bfa41-17c5-47b2-a560-fe2efbb2e4fb/2022-02-27%2017-03-13.mp4?AWSAccessKeyId=cloudconvert-production&Expires=1646090584&Signature=VyjiiTDMem6rnnxYt8t0MPR8Sm4%3D&response-content-disposition=attachment%3B%20filename%3D%222022-02-27%2017-03-13.mp4%22&response-content-type=video%2Fmp4

**Comment by osy** [comment]
Change number of cores and check the box to force multi threading but be warned it’s not going to be much faster.

**Reply by Its3rr0rsWRLD** [reply]
Its still 1 GHZ

**Reply by osy** [reply]
That number is fake. It can be slower or faster than 1GHz.

**Reply by Its3rr0rsWRLD** [reply]
Oh Ok. And that was fast lol.

**Reply by Its3rr0rsWRLD** [reply]
Is there any was I can make it so that when I use windows 11 in the VM it doesn't glitch. Like when dragging apps.
